composer.json

```diff
@@ -4,6 +4,7 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
+    "spryker/monitoring": "^2.0.1",
```

Comment:
Why is this required?

Reply:
Monitoring is made of 3 parts: web, console commands and record deployment. Console command logging is broken in Core, therefore it's needed to force the right fixed version for the New Relic monitoring to work out of the box.

Reply:
I changed to 2.0.0 cause I cannot force a dependency that is not there. So, for both new and old users, by then updating this package, they will automatically fall on the spryker/monitoring 2.0.1 (unless they change and specify 2.0.0 manually, but that is unlikely)